Translation corrections (#85)
07th-mod/higurashi-console-arcs: Update/some_006.txt

			NULL, "\"Natsumi-chan, that's not right.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310800037", 256, TRUE);
	OutputLine(NULL, "そんなこと、口に出しちゃいけないよ…？」",

> 			NULL, " You musn't say such things, you know...?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " You mustn't say such things, you know...?\"", Line_Normal);
```

	ClearMessage();

	DrawScene("background/aka1", 1000 );

//rおばーちゃんはさっきの私みたいに、笑顔に釣られて無理な笑顔を返してその場を丸く収めようとなんかしなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "おばーちゃんはさっきの私みたいに、笑顔に釣られて無理な笑顔を返してその場を丸く収めようとなんかしなかった。",

> 			NULL, "Unlike me, Grandma made no attempts to reciprocate my smile with a forced one of her own in order to ease the tension.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Unlike me, Grandma made no attempt to reciprocate my smile with a forced one of her own to ease the tension.", Line_Normal);
```

//rおばーちゃんのしなびた皮膚や、顔に刻まれている無数のしわが不気味に映って、全身に寒気が走る。…まるで、死人の顔のようにも見えて、心の底からおぞましい恐怖が押し寄せてきた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "おばーちゃんのしなびた皮膚や、顔に刻まれている無数のしわが不気味に映って、全身に寒気が走る。…まるで、死人の顔のようにも見えて、心の底からおぞましい恐怖が押し寄せてきた。",

> 			NULL, "Grandma's shriveled skin and the countless wrinkles engraved on her face eerily reflected through my eyest, sending a chill throughout my body. ...It looked like the face of a corpse, and it was absolutely terrifying.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Grandma's shriveled skin and the countless wrinkles engraved on her face eerily reflected through my eyes, sending a chill throughout my body. ...It looked like the face of a corpse, and it was absolutely terrifying.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>ＴＶアナ</color>", NULL, "<color=#f5e6d3>TV Announcer</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_tvann1015", 256, TRUE);
	OutputLine(NULL, "『さて、今回の有毒ガス事件ですが、雛見沢地区ではすでに村の守り神の祟りとして、伝承で予言されていたと言われています』",

> 			NULL, "\"Regarding the toxic gas incident, there are already whispers amongst residents of the Hinamizawa district that it is a curse enacted by the village's guardian deity as foretold in their traditions.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Regarding the toxic gas incident, there are already whispers amongst residents of the Hinamizawa district claiming it is a curse enacted by the village's guardian deity, as foretold in their traditions.\"", Line_Normal);
```
